win32/Lib/win32gui_struct.py

```diff
@@ -254,8 +254,8 @@ def EmptyMENUITEMINFO(mask=None, text_buf_size=512):
             | win32con.MIIM_STATE
             | win32con.MIIM_STRING
             | win32con.MIIM_SUBMENU
+            | win32con.MIIM_TYPE
         )
-        # Note: No MIIM_TYPE - this screws win2k/98.
```

Comment:
This needs to be tested, I have no idea of its actual effect. Just following the comment.